src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/CuDisbursementVoucherDocumentFixture.java

```diff
@@ -24,6 +32,9 @@
     public final String conferenceDestination;
     public final String nonEmployeeTravelerName;
     public final Integer nonEmployeeCarMileage;
+    public final Date dueDate;
+    public final Date invoiceDate;
```

Comment:
Date objects are mutable. If you want this fixture enum to remain immutable, you could create some getters that generate a new Date object when invoked, and within the fixture you could store the date information in some other immutable form (like leaving it as a String or using a Joda DateTime object).

Reply:
switched to joda datetime